src/tests/testSBuf.cc

```diff
@@ -1190,6 +1203,17 @@ TestSBuf::testSBufHash()
     }
 }
 
+void
+TestIoManip::testAsHex()
+{
+    SBufStream os;
+    os << asHex(0xa0);
+    CPPUNIT_ASSERT_EQUAL(SBuf("0xa0"), os.buf());
+    os.clearBuf();
+    os << asHex(0xa0,4);
+    CPPUNIT_ASSERT_EQUAL(SBuf("0x00a0"), os.buf());
```

Comment:
If you are going to test this, please do add a test where the number of digits higher than the limit and where the limit is zero.

```C++
CPPUNIT_ASSERT_EQUAL(SBuf("0x1234"), ToSBuf(asHex(0x1234).withLeadingZeros(2));
CPPUNIT_ASSERT_EQUAL(SBuf("0x4"), ToSBuf(asHex(0x00004).withLeadingZeros(0));
```

Reply:
sure, thanks